docs/_includes/doc-scroll.html

```diff
@@ -0,0 +1,28 @@
+<script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
```

Comment:
Is this easily solvable without javascript? I implemented it not to long ago because i thought it wasn't possible to do it without javascript. Maybe there is some better way so that we don't have the "?scroll=100" appending.